frontend-web/webclient/app/Grants/Editor.tsx

```diff
@@ -129,7 +133,7 @@ const defaultState: EditorState = {
             month: new Date().getMonth(),
             year: new Date().getFullYear(),
         },
-        durationInMonths: 12
+        durationInMonths: 1
```

Comment:
I don't think this was intentional, but let me know if it was.

```suggestion
        durationInMonths: 12
```